### PR TITLE
Add `users` to `ApiQueries`

### PR DESCRIPTION
**Pull request by japfohl**
add call to `/users/:userId` to `ApiQueries`

### CODE REVIEW

File: src/api/payloads/UsersPayload.ts [hunk 1]
```diff
@@ -0,0 +1,2 @@
+export interface UsersPayload {
+}
```
Comment (Crisfole) [inline src/api/payloads/UsersPayload.ts:2]: This is where you'd outline the structure of the return payload

File: src/api/args.ts [hunk 2]
```diff
@@ -50,3 +50,6 @@ export type VehicleQuery = Query & {
 export type ViolationsQuery = Query & {
   issued?: IntervalQuery;
 };
+
+export type UsersQuery = Query & {
+};
```
Comment (Crisfole) [inline src/api/args.ts:55]: No parameters accepted apart from default ones?

**Review by Crisfole — APPROVED** [review]
If you're not planning on adding to Payload LGTM

File: src/api/args.ts (hunk 2 above)
Comment (japfohl) [reply src/api/args.ts:55]: Correct.

File: src/api/payloads/UsersPayload.ts (hunk 1 above)
Comment (japfohl) [reply src/api/payloads/UsersPayload.ts:2]: Creating issue.

**Pull request merged**